.gitignore

```diff
@@ -2,4 +2,6 @@
 /node_modules/
 /package-lock.json
 /.nyc_output
-/coverage
\ No newline at end of file
+/coverage
+/benchmarks/node_modules/
+/benchmarks/package-lock.json
```

Comment:
should we add `/benchmarks/_results` to gitignore?

Reply:
I'm on a fence about it. Committing last run benchmarks might be actually helpful, people could view them in greater details then.

Reply:
fine by me